scripts/base/init-bare.bro

```diff
@@ -3710,11 +3710,36 @@ export {
 	## external harness and shouldn't output anything to the console.
 	const errors_to_stderr = T &redef;
 }
+
+module Fanout;
```

Comment:
I'm wondering if there's a better place for these options but I'm actually not sure right now where that would be. Ideally, we'd have a packet input framework or so, and they would become part of that. If they stay here, I think we'll need a different namespace than "Fanout", as that's lacking context for what it's about.